app/views/movies/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h3> Movie Information</h3>
+<h3> Title: <%= @movie.title %></h3>
+<h3> Year: <%= @movie.creation_year %></h3>
+<h3> Genre: <%= @movie.genre %></h3>
+<h5> Actors: <%= @movie.order_by_age %></h5>
+<h5> Actors Average Age: <%= @movie.avg_age %></h5>
+
+  <%= form_with url: "/movies/#{@movie.id}", method: :patch, data: {turbo: false} do |form| %>
+    <p><%= form.label :search_by_name, "Add an actor to this movie" %></p>
+    <%= form.text_field :search_by_name, value: params[:search_by_name] %>
```

Comment:
Instead of having a name field, the US was looking for a field where a user could enter the actor ID. Then, the ID would be passed to the controller action and could be used to create a MovieActor record